.drone.yml

```diff
@@ -3293,7 +3293,7 @@ steps:
         cat $WORKSPACE_DIR/curl_out.txt
         exit 1
       fi
-      curl $CREDENTIALS --fail -o /dev/null -F description="TODO" -F os="darwin" -F arch="amd64" -F "file=@$file" -F "sha256=$shasum" -F "releaseId=$product@$VERSION" "$RELEASES_HOST/assets";
+      curl $CREDENTIALS --fail -o /dev/null -F description="MacOS .pkg installer" -F os="darwin" -F arch="amd64" -F "file=@$file" -F "sha256=$shasum" -F "releaseId=$product@$VERSION" "$RELEASES_HOST/assets";
```

Comment:
Compared to the current downloads page, his has lost the `tsh client only` qualifier. We shortly discussed the ways to keep teleport/tsh packages distinct in https://github.com/gravitational/cloud/issues/1199 . Currently IMO the best course of action is have these separated by the `product` field (`teleport`, `teleport-ent`, `tsh`).